rero/rero-ils-ui
# circulation policy: improve editor

* Moves buttons to the top to harmonize with other editors.
* Corrects editor validation.
* Improves editor appearance.
* Closes rero/rero-ils#1365.
* Closes rero/rero-ils#1305.
* Adds id HTML attribute to circulation policy detail view for Cypress
  tests and improve its appearance.
* Removes useless code.
* Closes rero/rero-ils#1363.

Co-Authored-by: Alicia Zangger <alicia.zangger@rero.ch>

## Why are you opening this PR?

This is part of US1759

## Dependencies

My PR depends on `rero-ils#<xx>`'s PR(s):

https://github.com/rero/rero-ils/pull/1477

## How to test?

Login as a librarian and check the cipo editor.
Create a cipo with checkout duration = 0 days.
Create an item linked to this cipo.
Try to check it out, and then check it in for a patron that is also linked to this cipo.

## Code review check list
